pkg/platform/packet/packet.go

```diff
@@ -525,6 +526,37 @@ func (c *config) validateOSVersion() hcl.Diagnostics {
 	return diagnostics
 }
 
+// checkWorkerPoolLabelsAndTaints verifies that all worker pool labels
+// and taints are in correct format. Neither key nor value of labels
+// and taints map should have empty space in them.
+func (c *config) checkWorkerPoolLabelsAndTaints() hcl.Diagnostics {
+	var diagnostics hcl.Diagnostics
+
+	for _, w := range c.WorkerPools {
+		for k, v := range w.Labels {
+			if strings.Contains(k, " ") || strings.Contains(v, " ") {
```

Comment:
What if I put tab character in key or value? I think we should also validate that.